saleor/graphql/payment/mutations.py

```diff
@@ -28,7 +28,7 @@ class PaymentInput(graphene.InputObjectType):
             'Client-side generated payment token, representing customer\'s '
             'billing data in a secure manner.'))
     amount = Decimal(
-        required=True,
+        required=False,
```

Comment:
In that case, let's also change the description of that field: "Total amount of the transaction, including taxes and discounts. If no amount is provided, the checkout total will be used."